src/v3/package.json

```diff
@@ -37,7 +37,7 @@
     "build": "preact build --no-prerender --no-sw --dest ../../dist/dist/js",
     "clean-svg": "svgo -f src/img/authCoinIcons",
     "codegen": "yarn workspace @okta/okta-signin-widget codegen",
-    "dev": "preact watch --no-prerender --no-sw --port 3000",
+    "dev": "yarn codegen && preact watch --no-prerender --no-sw --port 3000",
```

Comment:
how about adding codegen to test:e2e?

Reply:
It's `dev` that actually breaks if `codegen` isn't run first, right? This seems like the right place IMO

Reply:
I was trying to decide the best place to put this. I went with this since the e2e test script runs the `dev` script, and if you do a fresh clone of the repo, trying to run `dev` will fail. I feel that this covers both situations well. Any objection?

Reply:
codegen takes time, and most probably we will need to run it once and it can be used for a long term, but we may restart local dev server several time. and I am ok with the current place, just feel it can become a bit annoying when restart happens